### DISCUSSION TITLE
DNS problem, please assist

### DISCUSSION THREAD

**Discussion by dhanadhyagroup** [postalserver/postal#2168] [Help with using Postal]
Hello Everybody, I have tried installing Postal mail server by following postal official installation documentation and this: https://inguide.in/build-postal-smtp-server-send-bulk-emails-ubuntu-18-04/ .

I have a static-ip form my ISP. I installed postal on i5 with 4GB RAM and 240GB SSD in my home server. In config file I gave LAN IP of 192.168.1.8, I can access web portal there. Then, I forwarded that ip to my public static ip at port 5000 & 25, check via some tools, I can access web portal and port 25 is open. I set hostname as postal.mydomain.site in config file. 

Now I have made changes to DNS server since 1.5 days in Hostinger, my domain provider. But it still shows that no records found. I have tried various things but in vain, although I can access web portal from my domain:5000 too. Pls assist, I am attaching relevant screenshots below.

Also, when I try to telnet my postal.mydomain.com, it shows that Name or service not known, but I can telnet to my public static ip to my server, both at port 25.

![WebUI on Domain 1](https://user-images.githubusercontent.com/69293267/193363635-de706905-8bc4-492c-923c-7feaea618db1.png)
![WebUI on Domain 2](https://user-images.githubusercontent.com/69293267/193363641-0462fdca-e5f4-49e3-a17e-d68b24382820.png)
![WebUI on Domain 3](https://user-images.githubusercontent.com/69293267/193363644-3556193d-748b-4683-9f49-08076eae5cd6.png)
![Telnet result](https://user-images.githubusercontent.com/69293267/193363646-3f6452f7-ab67-47d5-82eb-e40201382000.png)
![Config File](https://user-images.githubusercontent.com/69293267/193363648-f5ce7bbf-f147-4755-a6d8-c75759141305.png)
![DNS records Domain](https://user-images.githubusercontent.com/69293267/193363652-f78e3bed-47ea-48b8-9d27-bf2dc22a9f22.png)
![Router Port Forwarding](https://user-images.githubusercontent.com/69293267/193363655-460f993e-0c9e-4f22-8684-bbcfe16cba46.png)

**Comment by willpower232** [comment]
If you're running Postal "in house" as opposed to a cloud server provider, I can imagine your routing locally going a bit screwy so it might be worth testing the website works with a mobile device on 4G or someone elses wifi if you're able to do that.

This would also confirm that the ports are actually open according to your ISP as well.
